buildpipeline/pipeline.json

```diff
@@ -141,8 +140,7 @@
           "Parameters": {
             "PB_AdditionalBuildArguments": "/p:PortableBuild=true /p:SkipTests=true /p:NativeToolSetDir=C:\\tools\\clr",
             "PB_TargetArchitecture": "arm64",
-            "PB_PortableBuild": "true",
-            "PB_DistroRid": "win-arm64"
+            "PB_PortableBuild": "true"
```

Comment:
I don't believe this is the correct fix - it is not consistent with how we are doing non-ARM builds.

Instead, we need to be setting `$(PortableBuild)` to `true` in the ARM builds.